etc/help.txt

```diff
@@ -195,15 +195,26 @@ Tcl commands (Available in GUI or Batch console or Batch script):
                               : Uses alternate bitstream generation configuration files
 </openfpga>
 
----------------
---- Program ---
----------------
-   programmer <command> 
-     command                      :fpga_config, flash, fpga_status, list_devices
-     fpga_config <bitstream_file> : Program the device via JTAG
-     flash <bitstream_file>       : Program the flash via JTAG
-     fpga_status                  : Get the status of the device
-     list_devices                 : List all available devices
+------------------
+--- Programmer ---
+------------------
+   programmer <command>           : A programmer tool allows you to program and configure FPGA devices via JTAG 
+     command                      : fpga_config, flash, fpga_status, list_device, list_cable
+     fpga_config -c <cable name/index> ?-d <device_index>? <bitstream_file> : Program the FPGA device via JTAG
+         -c <cable_name/index>    : Specify the cable name or index
```

Comment:
Please make shift by 2 spaces here but not 4. Same for all other keys on this level

Reply:
Updated them to two spaces